simulator-gradle-plugin/build.gradle.kts

```
@@ -46,7 +47,7 @@ val generateVersionFile by
     doLast {
       val versionFile =
         File(generatedSourcesDir, "org/robolectric/simulator/gradle/generated/Version.kt")
```

With my previous comment, this would become something like:
```suggestion
      val versionFile = generatedSourcesDir.get()
        .file("org/robolectric/simulator/gradle/generated/Version.kt")
        .asFile
```